src/test/java/org/alfresco/rest/api/tests/client/data/NodeDirectAccess.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2016 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software. 
+ * If the software was purchased under a paid Alfresco license, the terms of 
+ * the paid license agreement will prevail.  Otherwise, the software is 
+ * provided under the following open source license terms:
+ * 
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.rest.api.tests.client.data;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.Serializable;
+
+import org.json.simple.JSONObject;
+
+/**
+ * Represents a direct access content url.
+ *
+ */
+public class NodeDirectAccess extends org.alfresco.rest.api.model.NodeDirectAccess implements Serializable, ExpectedComparison
+{
+
+    @Override
+    public void expected(Object o)
+    {
+        assertTrue("o is an instance of " + o.getClass(), o instanceof NodeDirectAccess);
+
+        NodeDirectAccess other = (NodeDirectAccess) o;
+
+        AssertUtil.assertEquals("contentUrl", getContentUrl(), other.getContentUrl());
+    }
+
+    public JSONObject toJSON()
```

Comment:
`toJSON()` and `parseEntity()` don't seem to be used anywhere.

Reply:
Good spot, I removed them.